src/main/java/com/wise23/chariteed/controller/PatientGeneratorController.java

```diff
@@ -0,0 +1,103 @@
+package com.wise23.chariteed.controller;
+
+import com.wise23.chariteed.QRCodeGenerator;
+import com.wise23.chariteed.model.Role;
+import com.wise23.chariteed.model.User;
+import com.wise23.chariteed.model.PatientGenerator;
+import com.wise23.chariteed.service.PatientService;
+import com.wise23.chariteed.service.UserService;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import org.hl7.fhir.r4.model.Patient;
+
+import com.google.gson.*;
+import com.google.zxing.WriterException;
+
+import java.util.Random;
+import java.io.IOException;
+import java.util.List;
+
+@Controller
```

Comment:
simply add here:
 @RequestMapping("/admin")
 or 
  @RequestMapping("/admin/dashboard")
  no need to write it everywhere here

Reply:
Done.